utils/render-chart-manifests.py

```diff
@@ -0,0 +1,64 @@
+import os
+import shutil
+import subprocess
+
+import yaml
+
+"""
+Simple script which renders the helm chart with the default values
+and removes some helm specific annotations. For convenience, we also
+generate a kustomization file.
+"""
+
+repo_root = (
```

Comment:
@ableuler I think it is not a bad idea to add a `if __name__ == "__main__"` just in case.

Reply:
👍 done